agent/agent_test.go

```diff
@@ -1430,8 +1460,12 @@ task_groups:
         params:
           script: echo hi
 `
-	s.setupRunTask(projYml)
-	s.tc.taskConfig.Task.TaskGroup = s.tc.taskGroup
+	p := &model.Project{}
+	_, err := model.LoadProjectInto(s.ctx, []byte(projYml), nil, "", p)
+	s.Require().NoError(err)
+	s.tc.taskConfig.Project = p
```

Comment:
It seems like replacing `setupRunTask` with `LoadProjectInto` + setting the project is unnecessary.

Reply:
reset it